contrib/fwupd.spec.in

```diff
@@ -278,6 +278,7 @@ fwupd wrapper for Qubes OS
 %endif
     -Dman=true \
     -Dbluez=true \
+    -Dpowerd=false \
```

Comment:
I think debian and arch builds also should avoid it.

Reply:
Probably snap too

Reply:
I removed it from debian(both), snap and freebsd, but I think we should leave it turned on for arch just so we have some CI coverage.